.editorconfig

```diff
@@ -1,7 +1,6 @@
 root = true
 
 [*]
-end_of_line = lf
```

Comment:
Atm, there is no reason to force `LF` on save in the working tree, as long as we force `LF` in the repo through the .gitattribute file.